docs/ci_runner_setup/linux/README.md

```diff
@@ -0,0 +1,78 @@
+## Linux CI runner setup
+
+This directory contains documentation and scripts about setting up a Linux CI Runner for [`ROCm`](https://github.com/ROCm) organization and used by [`TheRock`](https://github.com/ROCm/TheRock) repository.
+
+Note: you must have sufficient permissions to access [ROCm runner page](https://github.com/organizations/ROCm/settings/actions/runners)
+
+### Setup
+
+For brand new machines that do not that ROCm or Docker installed, please follow these steps. Otherwise, please skip to step 3.
+
+1. Install ROCm to the machine using `sudo ./rocm_install.sh`. This script will install ROCm 6.4 and AMD drivers for Ubuntu24, then it will reboot the system.
+
+   - Rebooting the system is required to load ROCm.
+   - If you have a different Linux distribution, follow [ROCm installation quick start guide](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/install/quick-start.html)
+   - <b>After reboot, please try `rocminfo` and `rocm-smi` to make sure ROCm is loaded and drivers are installed.</b> If there are issues, please try each command in `rocm_install.sh` instead.
+
+1. If docker is not installed, please run `sudo ./docker_install.sh`. This script will download docker for Ubuntu.
+
+1. After ROCm and Docker are installed, please run `sudo ./runner_setup_1.sh {IDENTIFIER}`. There may be multiple GPUs per system, so please add an identifier to make this runner unique and easily understood. Examples: gfx1201 GPU -> `sudo ./runner_setup_1.sh gfx1201-gpu-1`
+
+1. After the runner packages are there, please follow these steps and run the commands:
+
+   - Please retrieve token from [ROCm GitHub runner page](https://github.com/organizations/ROCm/settings/actions/runners/new?arch=x64&os=linux) in the `Configure` tab.
+   - Please add an unique identifying label for this CI runner. Example: Linux gfx1201 -> label `linux-gfx1201-gpu-rocm`. This is the label that will be used in workflows and will be shared amongst other identical machines.
+
+   ```
+   cd {IDENTIFIER_FROM_STEP_3}
+   ./config.sh --url https://github.com/ROCm --token {TOKEN} --no-default-labels --labels {LABEL}
+   ```
```

Comment:
nit: set the language for fenced code blocks when know to help with syntax highlighting (auto detection sometimes guesses correctly)
```suggestion
   ```bash
   cd {IDENTIFIER_FROM_STEP_3}
   ./config.sh --url https://github.com/ROCm --token {TOKEN} --no-default-labels --labels {LABEL}
   ```
```